apps/full-site-editing/full-site-editing-plugin/full-site-editing/class-full-site-editing.php

```diff
@@ -312,7 +312,7 @@ public function enqueue_script_and_style() {
 		wp_enqueue_style(
 			'a8c-full-site-editing-style',
 			plugins_url( 'dist/' . $style_file, __FILE__ ),
-			array(),
+			'wp-edit-post',
```

Comment:
This is to ensure that our styles are loaded after the Gutenberg styles for the editor, making easier to override a specific style.